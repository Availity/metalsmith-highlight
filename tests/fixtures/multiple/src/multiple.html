<code class="language-json">
{
  "name": "metalsmith-prism",
  "description": "Provides syntax highlighting"
}
</code>

<code class="language-ruby">
plugin = { name: "metalsmith-prism" }
puts "Good day to you, from #{plugin.name}"
</code>

<code class="language-bash">
$ echo "Hello World"
</code>
