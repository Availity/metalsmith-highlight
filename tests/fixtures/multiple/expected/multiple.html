<code class="language-json">
<span class="token punctuation">{</span>
  <span class="token keys">&quot;name&quot;</span><span class="token punctuation">:</span> &quot;metalsmith-prism&quot;<span class="token punctuation">,</span>
  <span class="token keys">&quot;description&quot;</span><span class="token punctuation">:</span> &quot;Provides syntax highlighting&quot;
<span class="token punctuation">}</span>
</code>

<code class="language-ruby">
plugin <span class="token operator">=</span> <span class="token punctuation">{</span> name<span class="token punctuation">:</span> <span class="token string">&quot;metalsmith-prism&quot;</span> <span class="token punctuation">}</span>
puts <span class="token string">&quot;Good day to you, from <span class="token interpolation"><span class="token delimiter tag">#{</span>plugin<span class="token punctuation">.</span>name<span class="token delimiter tag">}</span></span>&quot;</span>
</code>

<code class="language-bash">
$ <span class="token keyword">echo</span> <span class="token string">&quot;Hello World&quot;</span>
</code>
